joomla/joomla-cms
# [4.0] Cassiopeia - fixes fluid width option

Pull Request for Issue #26887 .

### Summary of Changes
Fixes the fluid with option in cassiopeia template settings.


### Testing Instructions
Apply patch and run `node build.js --compile-css` to update the changed SCSS.

Enable 'Fluid Width' option in template settings. Check frontend.


### Before
![image](https://user-images.githubusercontent.com/2803503/71079686-a15ed300-2183-11ea-9887-ec5cee401f58.png)



### After
![image](https://user-images.githubusercontent.com/2803503/71079650-8f7d3000-2183-11ea-8bf8-b043c905d963.png)



### Documentation Changes Required
